haas/api.py

```diff
@@ -1073,6 +1089,7 @@ def show_node(nodename):
                                      attachment.network.label)
                                     for attachment in n.attachments]),
                   } for n in node.nics],
+        'metadata': {m.label: m.value for m in node.metadata}
```

Comment:
The README says that HIL requires python 2.7.

I'd rather we put effort into supporting python 3 over 2.6 :-)